src/less/main.less

```diff
@@ -0,0 +1,10 @@
+// App.less
+@import "../components/button/button.less";
```

Comment:
you should probably put all the less files inside the "less" folder to make this easier